packages/perseus-core/src/parse-perseus-json/perseus-parsers/free-response-widget.ts

```diff
@@ -9,5 +9,10 @@ export const parseFreeResponseWidget: Parser<FreeResponseWidget> = parseWidget(
     constant("free-response"),
     object({
         question: string,
+        scoringCriteria: array(
+            object({
+                text: string,
+            }),
+        ),
```

Comment:
Have any free-response widgets been created in production? Any existing free-response widgets won't have `scoringCriteria`, so this parser will fail for them.

If you need to be compatible with existing data, I'd suggest defaulting to an empty array:

```ts
scoringCriteria: defaulted(
    array(
        object({
            text: string,
        }),
    ),
    () => [],
),
```

And you'd also want to add a regression test to `parse-perseus-json/regression-tests/item-data` to demonstrate the need for the default.

Reply:
There are no FRW's (Free-Response Widget) in prod. The feature branch that we're merging into will be the introduction of this widget.